src/main/java/com/neuvector/model/NVScanner.java

```diff
@@ -38,14 +40,16 @@ public NVScanner(String nvScannerImage, String nvRegistryURL, String nvRegistryU
      * @param nvMountPath  The mount path mapping to the path inside the NeuVector Scanner container. It is the path to store the scan result. It is optional. If you don't pass in <code>nvMountPath</code>, it will use the default path "/var/neuvector"
      * @param bindMountShared Indicates whether the bind mount content needs to be shared. Using True when it's needed. If null is received, the default value, which is False, will be used.
      */
-    public NVScanner(String nvScannerImage, String nvRegistryURL, String nvRegistryUser, String nvRegistryPassword, String nvMountPath, Logger log, Boolean bindMountShared){
+    public NVScanner(String nvScannerImage, String nvRegistryURL, String nvRegistryUser, String nvRegistryPassword, String nvMountPath, Logger log, Boolean bindMountShared, String runtime, String socketMapping){
```

Comment:
Can you add javadocs for the new params as well?

Reply:
Done in 232194fe45841f1e6ee155f0d3a86254cc1c8291 and c6511d28c303e030b3c334f7adc6621d972ca7e7